plugins/ci/pkg/ci/trivy.go

```diff
@@ -131,12 +140,23 @@ func downloadMissingImages(folderPath string, dockerImages []trivymodels.DockerI
 			continue
 		}
 		rc := registryCredentials.FindCredentialForImage(image.Name)
-		_, err := downloadImageViaSkopeo(commands.ExecWithMessage, folderPath, image.Name, rc)
+		output, err := downloadImageViaSkopeo(commands.ExecWithMessage, folderPath, image.Name, rc)
 		if err != nil {
-			return nil, err
+			allErrs = multierror.Append(allErrs, fmt.Errorf("%v: %s", err, output))
+		} else {
+			image.PullRef = clearString(image.Name)
+			refLookup[image.Name] = image.PullRef
+		}
+	}
+	if len(allErrs.Errors) > 0 {
+		return ciutil.ReverseMap(refLookup), models.ScanErrorsReportResult{
+			ErrorMessage: allErrs.Error(), // keep multiple errors combined into a single error / action item
+			ErrorContext: "downloading missing images to be scanned by trivy",
+			Kind:         "InternalOperation",
+			Remediation:  "Please inspect the action item description for signs of why downloading images may have failed.",
+			ResourceName: "DownloadMissingImages",
+			Filename:     filepath.Clean(folderPath), // clean() removes potential trailing slash
```

Comment:
Is it reasonable to use the general images directory as specified in fairwinds-insights.yaml, as the CI filename for errors encountered when downloading missing images?

Reply:
It could be either an image error or a writing error, ideally I would use `fairwinds-insights.yaml` for `dockerImages` downloading errors, however, as we are combining multiple errors into one AI there is not really a right answer to the filename.

I guess `folderPath` is reasonable (more so when it is a writing issue)